plugins/base/src/test/kotlin/model/PropertyTest.kt

```diff
@@ -138,6 +138,29 @@ class PropertyTest : AbstractModelTest("/src/main/kotlin/property/Test.kt", "pro
         }
     }
 
+    @Test
```

Comment:
Could you also update the previous test (`propertyOverride`) to include assertion about DRI?